dialogue-serde/src/main/java/com/palantir/conjure/java/dialogue/serde/DefaultConjureRuntime.java

```diff
@@ -67,8 +73,19 @@ public Clients clients() {
 
         private final List<WeightedEncoding> encodings = new ArrayList<>();
 
+        private CaffeineSpec cacheSpec = DEFAULT_SERDE_CACHE_SPEC;
+
         private Builder() {}
 
+        /**
+         * Specify the serializer and deserializer cache configuration.
+         */
+        @CanIgnoreReturnValue
+        public Builder serializerCacheSpecification(String specification) {
+            cacheSpec = CaffeineSpec.parse(specification);
+            return this;
+        }
```

Comment:
Thinking about this a bit more, I'm not sure we want to expose this as API. It controls an internal detail that should be tuned to help our consumers, but not something they should tune themselves or depend upon because it could change.

Reply:
removed